MapboxNavigation/Locale.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+extension Locale {
+    static var preferredLocalLanguageCountryCode: String {
+        let firstBundleLocale = Bundle.main.preferredLocalizations.first!
+        let bundleLocale = firstBundleLocale.components(separatedBy: "-")
+        
+        if bundleLocale.count > 1 {
+            return firstBundleLocale
+        }
+        
+        let countryCode = NSLocale.Key.countryCode
```

Comment:
Nit: inline this expression. It’s a constant, so there’s no advantage to breaking it out. In fact, since `NSLocale.object(forKey:)` takes an `NSLocale.Key`, which is an enumeration, you can pass in just `.countryCode`.